src/plays/lofi-player/styles.css

```diff
@@ -0,0 +1,31 @@
+/* enter stlyes here */
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap');
+
+*{
```

Comment:
Do not override with * as it will impact other part of the UI.

Please create a unique class that related to your play and use that as prefix for all the styles in this file.

Reply:
done